app/routes/events/view/edit/basic-details.js

```diff
@@ -1,13 +1,16 @@
 import Ember from 'ember';
 import EventWizardMixin from 'open-event-frontend/mixins/event-wizard';
 
-const { Route } = Ember;
+const { Route, RSVP } = Ember;
 
 export default Route.extend(EventWizardMixin, {
   titleToken() {
     return this.l10n.t('Basic Details');
   },
   model() {
-    return this.modelFor('events.view.edit');
+    let data = this.modelFor('events.view.edit');
+    data.copyright = this.getOrCreate(data.event, 'copyright', 'event-copyright');
+    data.tax = this.getOrCreate(data.event, 'tax', 'tax');
```

Comment:
@niranjan94 This results in rejection of promise even if the tax exists for the event. Throws an error
```
Assertion Failed: You cannot add a record of modelClass 'undefined' to the 'event.tax' relationship (only 'tax' allowed)
```

Works perfectly for copyright & speakers call. Can you please help?